types/node/util.d.ts

```diff
@@ -83,24 +83,27 @@ declare module "util" {
         fn: (arg1: T1, arg2: T2, arg3: T3, arg4: T4, arg5: T5, arg6: T6) => Promise<TResult>
     ): (arg1: T1, arg2: T2, arg3: T3, arg4: T4, arg5: T5, arg6: T6, callback: (err: NodeJS.ErrnoException | null, result: TResult) => void) => void;
 
+    // The value type here is a "poor man's `unknown`". When these types support TypeScript
+    // 3.0+, we can replace this with `unknown`.
+    type PoorMansUnknown = {} | null | undefined;
```

Comment:
I thought about avoiding the need for this by making this change to the 3.1+ types, but I think we ought to provide this bug fix to all users.

Reply:
Maybe move the `PoorMansUnknown` to NodeJS namespace in global.d.ts and use it also at other locations.